lua/nvim-tree/actions/find-file.lua

```diff
@@ -13,17 +14,35 @@ function M.fn(fname)
   end
   running[fname] = true
 
+  -- always match against the real path
+  local fname_real = uv.fs_realpath(fname)
+  if not fname_real then
+    return
+  end
+
   local i = view.is_root_folder_visible() and 1 or 0
   local tree_altered = false
 
   local function iterate_nodes(nodes)
     for _, node in ipairs(nodes) do
       i = i + 1
-      if node.absolute_path == fname then
-        return i
+
+      local stat, _ = uv.fs_stat(node.absolute_path)
+      if not stat then
+        break
+      end
+      local real_path, _ = uv.fs_realpath(node.absolute_path)
+      if not real_path then
+        break
       end
 
-      local path_matches = node.nodes and vim.startswith(fname, node.absolute_path .. utils.path_separator)
+      -- match against node absolute and real, for the case of symlinks, which will differ
+      if node.absolute_path == fname_real or real_path == fname_real then
+        return i
+      end
+      local abs_match = vim.startswith(fname_real, node.absolute_path .. utils.path_separator)
+      local real_match = vim.startswith(fname_real, real_path .. utils.path_separator)
```

Comment:
We could keep a "real path" on the node, however that seems too complicated.

Reply:
There is a `link_to` attributes for symlinks on node, maybe this is what you would like to use here ?

Reply:
> There is a `link_to` attributes for symlinks on node, maybe this is what you would like to use here ?

Goot thinking. I'll rework on the weekend.

Reply:
It is now simpler, thank you.